nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/SchemaManagerImpl.java

```diff
@@ -576,6 +580,11 @@ protected void recomputeDocumentTypes() {
             }
         }
 
+        // special document types (excluded from copy)
+        specialDocumentTypes = allDocumentTypes.stream()
```

Comment:
I think this should be `dtds.values().stream()` actually, we want to do this on the effective descriptors with overrides and merges taken into account.